### DISCUSSION TITLE
Issue install bare-metal V4.9.0 & 4.13.0

### DISCUSSION THREAD

**Discussion by JeanNNI** [okd-project/okd#1703] [Q&A]
Hello,
I am trying to install OKD on my machine using version 4.9.0 using the command "oc create install-config" and I can't get the "bare-metal" section, I tried using a more recent version 4.13.0 to see if I can get it, but still nothing... What is the reason for this?

**Comment by vrutkovs** [comment]
>I can't get the "bare-metal" section

You mean there is no section like this in the installer?

**Reply by JeanNNI** [reply]
That's right, it doesn't appear in either version.

**Reply by vrutkovs** [reply]
Ah, default installer is not built with baremetal IPI support. You can extract it from `baremetal-installer` image:
```
oc adm release extract --command=openshift-baremetal-install --from=quay.io/openshift/okd:4.13.0-0.okd-2023-08-04-164726
```

**Reply by JeanNNI** [reply]
I tried get this command, but no found baremetal in dropmenu.
![None-openshift](https://github.com/okd-project/okd/assets/110637117/7de1d7e1-e131-419a-8777-f4ac2198c840)
![NotfoundBare-Metal](https://github.com/okd-project/okd/assets/110637117/555a7774-726b-44b3-9e7a-0248173ef07b)

**Reply by vrutkovs** [reply]
works here:
```
$ oc adm release extract --command=openshift-baremetal-install --from=quay.io/openshift/okd:4.13.0-0.okd-2023-08-04-164726
Warning: the default reading order of registry auth file will be changed from "${HOME}/.docker/config.json" to podman registry config locations in the future version of oc. "${HOME}/.docker/config.json" is deprecated, but can still be used for storing credentials as a fallback. See https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md for the order of podman registry config locations.
Warning: the default reading order of registry auth file will be changed from "${HOME}/.docker/config.json" to podman registry config locations in the future version of oc. "${HOME}/.docker/config.json" is deprecated, but can still be used for storing credentials as a fallback. See https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md for the order of podman registry config locations.
$ ./openshift-baremetal-install create cluster
? SSH Public Key /var/home/vrutkovs/.ssh/id_rsa.pub
? Platform  [Use arrows to move, type to filter, ? for more help]
  alibabacloud
  aws
  azure
> baremetal
```

**Reply by JeanNNI** [reply]
Hi, ur command worked... But i tried "install" baremetal and I have seen that it gives me an error and wants to redirect the ip to an amazon page IP :15.197.204.X.

? SSH Public Key <none>
? Platform baremetal
? Provisioning Network Managed
? Provisioning Network CIDR 172.22.0.0/24
? Provisioning bridge br0
? Provisioning Network Interface br0
? External bridge baremetal
? Add a Host: control plane
? Name master
? BMC Address http://192.168.10.15
? BMC Username root
? BMC Password ****
? Boot MAC Address xx:xx:xx
? Add another host? No
? Base Domain testlab.com
? Cluster Name labcluster
? The container registry pull secret for this cluster, as a single line of JSON (e.g. {"auths": {...}}).

You can get this secret from https://console.redhat.com/openshift/install/pull-secret
? Pull Secret *********************************************************
FATAL failed to fetch Metadata: failed to fetch dependency of "Metadata": failed to fetch dependency of "Cluster ID": failed to generate asset "Install Config": invalid install config: [platform.baremetal.apiVIPs: Invalid value: "15.197.204.X": VIP for API must not be one of the Ingress VIPs, platform.baremetal.apiVIPs: Invalid value: "15.197.204.X": IP expected to be in one of the machine networks: 10.0.0.0/16, platform.baremetal.ingressVIPs: Invalid value: "15.197.204.X": IP expected to be in one of the machine networks: 10.0.0.0/16, platform.baremetal.Hosts: Required value: not enough hosts found (1) to support all the
